python/pyspark/ml/classification.py

```diff
@@ -16,19 +16,24 @@
 #
 
 import operator
+import sys
```

Comment:
Both `sys` and `uuid` are used in the class, but aren't being imported.

uuid: https://github.com/apache/spark/blob/master/python/pyspark/ml/classification.py#L3027
sys: https://github.com/apache/spark/blob/master/python/pyspark/ml/classification.py#L3421